api/server/agent.go

```diff
@@ -11,10 +11,11 @@ package api
 import (
 	"encoding/json"
 	"fmt"
-	"github.com/Mierdin/todd/agent/defs"
-	"github.com/Mierdin/todd/db"
```

Comment:
TODO this has been removed entirely in this patch, so I'm assuming this means the tapi.tdb provides this functionality indirectly?